hw/hw_0001/index.3.js

```diff
@@ -1,17 +1,26 @@
 let currencies = [
-    {name: "hrn", cost: 1},
-    {name: "usd", cost: 25.2},
-    {name: "eur", cost: 30.1},
+  {shortname: "AUD", name: "Австралійський долар", cost: 16.155},
```

Comment:
We need to add an additional field with available amount of corresponding currency smth like `amount`

Reply:
Ok.
- add `amount`
- Now the program displays the `amount `of currency multiplied by `rate` and the total purchase price.